eng/jacoco-test-coverage/pom.xml

```diff
@@ -225,6 +225,11 @@
       <artifactId>azure-json</artifactId>
       <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-json;current} -->
     </dependency>
+    <dependency>
```

Comment:
does it have to be a new package? Could it be done as a part of `azure-core-serializer-json-gson` ?

Reply:
I think this could be moved into `azure-core-serializer-json-gson` but for now I want to keep it in `azure-json-gson`